### PR TITLE
GLUE on TPU

### PR DESCRIPTION
**Pull request by LysandreJik**
This takes advantage of the pytorch 1.3 XLA implementation to fine-tune GLUE on a TPU.

MRPC fine-tuning in 3 epochs + evaluation takes a total of 6 minutes and 30 seconds.

### CODE REVIEW

**Comment by thomwolf** [comment]
LGTM!

**Comment by guozhiyu** [comment]
It is possible to save/load when on TPU, just move them to CPU and save them:

https://github.com/pytorch/xla/blob/master/API_GUIDE.md#saving-and-loading-xla-tensors
